lib/Renard/Curie/Data/PDF.pm

```diff
@@ -2,43 +2,101 @@ use Modern::Perl;
 package Renard::Curie::Data::PDF;
 
 use Capture::Tiny qw(capture_stdout);
+use XML::Simple;
+use Path::Tiny;
+use Alien::MuPDF 0.004;
 
-sub mudraw_get_pdf_page_as_png {
-	my ($pdf_filename, $pdf_page_no) = @_;
+BEGIN {
+	our $MUTOOL_PATH =  path(Alien::MuPDF->dist_dir, qw(bin mutool));
+}
 
+sub _call_mutool {
```

Comment:
I created a single function that takes arguments and passest them on to `mutool`.